File: backend/src/main/kotlin/com/example/backend/dtos/UserDto.kt
package com.example.backend.dtos

import org.bouncycastle.util.test.FixedSecureRandom.BigInteger
import java.math.BigDecimal
import java.time.LocalDate
import java.time.LocalDateTime

data class UserNewDto(
  val nom: String?,
  val prenom: String?,
  val email: String?,
  val fonction: String?,
  val reduction: String?,
  val telephone: String?,
  val reductionMax: String?,
  val reductionFait: String?,
  val id: String?
)

File: backend_py/pharmaPython/app/schemas/user_dto.py
from pydantic import BaseModel
from typing import Optional

class UserNewDto(BaseModel):
  nom: Optional[str] = None
  prenom: Optional[str] = None
  email: Optional[str] = None
  fonction: Optional[str] = None
  reduction: Optional[str] = None
  telephone: Optional[str] = None
  reductionMax: Optional[str] = None
  reductionFait: Optional[str] = None
  id: Optional[str] = None
